kernels/ZendEngine2/string.c

```diff
@@ -305,19 +311,25 @@ void zephir_camelize(zval *return_value, const zval *str) {
 }
 
 /**
- * Convert a camelized to a dash/underscored texts
+ * Convert a camelized to a dash/underscored texts (an optional delimiter can be specified)
  */
-void zephir_uncamelize(zval *return_value, const zval *str) {
+void zephir_uncamelize(zval *return_value, const zval *str, const zval *delimiter) {
 
 	unsigned int i;
 	smart_str uncamelize_str = {0};
-	char *marker, ch;
+	char *marker, ch, delim;
 
 	if (Z_TYPE_P(str) != IS_STRING) {
```

Comment:
I think you mean something else or I've misunderstood :) Because it's pointless

```c
	if (Z_TYPE_P(str) != IS_STRING || Z_STRLEN_P(str) > 1) {
		zend_error(E_WARNING, "Invalid arguments supplied for uncamelize()");
		return;
	}
```